bika/lims/browser/worksheet/views/analyses.py

```diff
@@ -439,3 +447,42 @@ def get_slot_header(self, item):
 
         pos_text += "</table>"
         return pos_text
+
+    def fill_remarks_cells(self, items):
+        """Gets the Remarks from the parent AR for each slot.
+        :param items: dictionary with items to be rendered in the list
+        """
+        filled_slots = []
+        for item in items:
```

Comment:
If you call this one from inside `folderitem` (and pass in the item, together with the object) you will not need to iterate through all items and you'll also be able to call `get_slot_remarks` passing the object instead of the item, that imo, would be better